packages/authentication/docs/strategies/basic-auth.md

```diff
@@ -0,0 +1,30 @@
+You could find the `AuthenticationStrategy` interface in file
+[authentication-strategy.md](./docs/authentication-strategy.md)
+
+```ts
+import {Request} from '@loopback/rest';
+
+class BasicAuthenticationStrategy implements AuthenticationStrategy {
+  options: object;
+  constructor(
+    @inject(AUTHENTICATION_BINDINGS.SERVICES.USER) userService: UserService,
```

Comment:
Maybe `AUTHENTICATION_BINDINGS.USER_SERVICE`? Let's constrain the namespace to be at most one level.

Reply:
makes sense